COSC481W-2024Fall/MLite
# Nicholas fiori rails deployments inference form page

Features:

- Added a deployment inference page
- Added dummy fields

Testing:

- Go to deployments.
- Select run inference for a deployment created previously.
- Enter an integer for the age field.
- Enter a number for the height field; the smallest decimal point you can reach is a hundredth.
- Choose whether or not to check the checkbox for the Do you have a car field.
- Choose a color from the drop down menu of the color field
- Hit the submit button below.
- A Inference Result should appear at the bottom of the page, indicating your choices.

Issue #27
